magento/magento2
# Issue #23522 UPS shipping booking and label generation gives error when shipper's street given more than 35 chars

### Description (*)


### Fixed Issues (if relevant)

1. magento/magento2#23522: Issue title

### Manual testing scenarios (*)

1. Configure Origin in In shipping settings and address should be 
(Street1 : "Xth Floor, The Great Persis,", Street2: "D95 Road No 91, LIPC, Anpheri (G)"), 
the logic is to make total characters by combining both street should be more than 35 chars.
2. Place and order with any Germany city address by selecting any of the available UPS shipping method.
3. Now create shipment for an order. While creating shipment check the checkbox "create shipping label" and proceed.
4. After filling details in create packages form do click save

### Questions or comments


### Contribution checklist (*)
